diff --git a/node_modules/fetch-blob/from.js b/node_modules/fetch-blob/from.js
index 9eaf8bf..fdff56e 100644
--- a/node_modules/fetch-blob/from.js
+++ b/node_modules/fetch-blob/from.js
@@ -1,6 +1,13 @@
 import { statSync, createReadStream, promises as fs } from 'node:fs'
 import { basename } from 'node:path'
-import DOMException from 'node-domexception'
+const DOMException = typeof globalThis.DOMException === 'function'
+  ? globalThis.DOMException
+  : class DOMException extends Error {
+    constructor (message = '', name = 'Error') {
+      super(message)
+      this.name = name
+    }
+  }
 
 import File from './file.js'
 import Blob from './index.js'
